olive/model/handler/mixin/hf_config.py

```diff
@@ -82,14 +66,6 @@ def get_hf_model_tokenizer(self):
             self._get_model_path_or_name(), **self.hf_config.get_loading_args_from_pretrained()
         )
 
-    def save_hf_model_tokenizer(self, output_dir: str, **kwargs):
-        if self.hf_config is None:
-            raise ValueError("HF model_config is not available.")
-        if not Path(output_dir).is_dir():
-            raise ValueError("Expecting a directory as input.")
-
-        save_hf_model_tokenizer(self.get_hf_model_tokenizer(), output_dir, **kwargs)
-
     def save_metadata_for_token_generation(self, output_dir: str, **kwargs):
```

Comment:
it seems only this save_metadata_for_token_generation need to be public and all other new added files can be private. In the mean  time, please remember update the document section in line 32

Reply:
Those are all generic get functions (get_hf_config, get_hf_generation_config, etc) with no internal bookkeeping to maintain. I don't see a point in making those private.  I did remove the save_xxx functions since they weren't used anymore.

Reply:
From the time being, all method in Olive model mixins doesn't start with underscore will be public. You can see https://github.com/microsoft/Olive/blob/main/olive/model/handler/mixin/hf_config.py#L162

Reply:
Done!